pkg/hostagent/opflex.go

```diff
@@ -110,9 +110,13 @@ func (agent *HostAgent) discoverHostConfig() (conf *HostAgentNodeConfig) {
 			conf.UplinkIface = parent.Attrs().Name
 			conf.VxlanAnycastIp = anycast.String()
 			conf.OpflexPeerIp = peerIp.String()
-			return
 		}
 	}
+	intf, err := net.InterfaceByName(conf.UplinkIface)
+	if err == nil {
+		conf.UplinkMacAdress = intf.HardwareAddr.String()
+		agent.log.Info("Hardware address is ", intf.HardwareAddr)
+	}
```

Comment:
Should we proceed further if we don't know the mac address?

Reply:
return is added